_topic_map.yml

```diff
@@ -2081,6 +2088,33 @@ Topics:
 - Name: Troubleshooting
   File: troubleshooting-3-4
 ---
+Name: Migrating from OKD 3 to 4
```

Comment:
@apinnick The problem is when "OpenShift Container Platform" is hard coded in the topic map. Those instances should be OKD for OKD. To resolve this, we have to use the "Distros" field to show different ones per product.

Usually we can just duplicate and slap a Distros tag when it's in an assembly name. But for MTC, the product name is in the actual folder name. So I had to copy the entire book/section, and make one for the OCP distro and one for the OKD distro. Not idea, because you'll need to remember to update both for any future updates to the section in the topic map. But can you please review?

@vikram-redhat You aren't aware of any other solution here, are you? The only other option I can think of is to not use the product name in the bucket name, but I assume that's not a good option.

Reply:
I reviewed the previews and they look OK. Sorry about creating the need for a workaround. I had no idea that the product name in that context would be problematic. In theory, I could remove "ocp" from the folder name. Let me know if you think that is a viable solution.

Reply:
@apinnick thanks, and no worries! And it's up to you on whether you want to remove OCP/OKD from the folder name. If you have an alternative name that you think would still be good, then that would probably be easier on you in the future. (To prevent errors where you or someone updates one block but forgets to do the other one.

If you want to go that route, I can pull the change from this PR. Just let me know what you want to do!

Reply:
@bergerhoffer If it is not critical, I would prefer to deal with this at a later date. The only alternative I can think of is to remove OCP, so it would be "migrating_from_3_to_4", which seems as good as any. 

Is there a note about not using the product name in top-level folders in the OCP doc guidelines or manual? If not, perhaps someone could add that. 

Might also be worth adding to the peer review checklist, although that's lower priority. I would assume that top-level folders are not created very frequently. 

I'm not sure what you mean about "To prevent errors where you or someone updates one block but forgets to do the other one". I'll ask you in Slack.

In the meantime, go ahead with whatever solution you think best.

Reply:
@apinnick Agree it would be better in the long run to adjust the name as opposed to maintaining two sections. I'm happy to make the adjustments in this PR if you let me know what you'd like to go with. 

For now, I've updated it from "Migrating from OpenShift Container Platform 3 to 4" to "Migrating from version 3 to 4", but let me know if you have a different title you'd like.

Note that I did have to duplicate the two assemblies (one w/ a distro of OCP and one w/ a distro of OKD). that should be easier to keep up with (and is what we do a few other places in the topic map). But let me know if you'd prefer to adjust those names as well or anything.

Reply:
LGTM

Reply:
I'd say that this is sufficient for now (we do this several other places), but if you'd rather adjust the assembly names, feel free to consider that in a future update. Thanks for your help!